io.openems.edge.ess.sinexcel/src/io/openems/edge/ess/sinexcel/Config.java

```diff
@@ -29,8 +29,11 @@
 
 	@AttributeDefinition(name = "Battery target filter", description = "This is auto-generated by 'Battery-ID'.")
 	String Battery_target() default "";
-
-	@AttributeDefinition(name = "Start and standby", description = "Turn ON and turn OFF the Inverter")
+	
+	@AttributeDefinition(name = "slow float charge", description = "slow float charge")
+    int slowFloatCharge() default 4739;
```

Comment:
Eclipse autoformat. What does this config parameter do? Name and Description are the same :-(